template/lib.typ

```diff
@@ -1,5 +1,16 @@
+#import "@preview/droplet:0.3.1": dropcap
 #let etsi_color = rgb(83, 16, 12)
 
+#let FirstLetter(text) = {
+  dropcap(
+  height: 3,
+  gap: 4pt,
+  hanging-indent: 0em,
+  overhang: 0pt,
+  font: ("Times New Roman", "Tinos"),
+text)
+}
+
```

Comment:
Isn't this unused?

Reply:
no its left for the user as a function to use. It allows you to get the dropcap at the beginning of a 
paragraph

![image](https://github.com/user-attachments/assets/31d0c1fd-cc28-4f56-b392-c0fa1c41db04)

Reply:
To organize a little the changes Im gonna change this and the tiltes changes to another PR